### DISCUSSION TITLE
Zoom to mouse or window center.

### DISCUSSION THREAD

**Discussion by moon47usaco** [zanllp/infinite-image-browsing#272] [Ideas]
Currently it zooms to the center of the image.

This is bad because if you are zoomed in and pan over, another zoom in or out will re center and likely take you away from the current fucus. Especially when looking at xyz plots. =\

Having the zoom center at the mouse would be ideal but if that is problematic, just using the current window center for the zoom would work better.

BTW: Thank you sooooo much for adding the image info (name, size & res) to the info window... =]
How many times have I searched for the image I had open. Thank you....... =]

R

**Comment by zanllp** [comment]
Implementing this feature is quite complex, and while I would like to see it in the software, it seems that the implementation cost is quite high at the moment. However, I will try to work on it when I have some free time. In the meantime, you may want to check out our previous discussion on this topic at https://github.com/zanllp/sd-webui-infinite-image-browsing/issues/137. 

Thanks for bringing this to our attention, and we appreciate your feedback. Let us know if you have any further suggestions or comments.
